src/main/java/com/softlayer/api/RestApiClient.java

```diff
@@ -41,6 +41,18 @@
     static {
         HEADERS = Collections.singletonMap("SoftLayer-Include-Types", Collections.singletonList("true"));
     }
+
+    /**
+     * A list of service methods that do not have to be added to the REST URL.
+     * createObjects is supposed to work, but does not.
+     */
+    private static final List<String> IMPLICIT_SERVICE_METHODS = Arrays.asList(
```

Comment:
I didn't see this before, but it should be noted a `Set` is usually better for these types of things.